### PR TITLE
Include maintainers in release notes

### PR DESCRIPTION
**Pull request by murrant**
Use GraphQL to pull in the exact data we need

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: LibreNMS/Util/GitHub.php [hunk 1]
```diff
@@ -167,33 +213,42 @@ public function buildChangeLog()
...
+     *
+     * @param array $user
+     * @param string $type
+     */
+    private function recordUserInfo($user, $type = 'changelog_users') {
+        $user_count = &$this->$type;
+
+        $user_count[$user['login']] = isset($user_count[$user['login']])
+            ? $user_count[$user['login']] + 1
+            : 0;
```
Comment (Jellyfrog) [inline LibreNMS/Util/GitHub.php]:
```suggestion
            : 1;
```

File: LibreNMS/Util/GitHub.php [hunk 2]
```diff
@@ -208,14 +263,16 @@ public function formatChangeLog()
...
-            foreach ($this->changelog_users as $user => $count) {
-                $tmp_markdown .= "  - $user ($count)" . PHP_EOL;
-            }
+            $tmp_markdown .= $this->formatUserList($this->changelog_users);
         }
 
         $tmp_markdown .= PHP_EOL;
 
+        if (!empty($this->changelog_mergers)) {
+            $tmp_markdown .= "Thanks to maintainers that helped merge pull requests this month: " . PHP_EOL . PHP_EOL;
```
Comment (Jellyfrog) [inline LibreNMS/Util/GitHub.php]:
```suggestion
            $tmp_markdown .= "Thanks to maintainers that helped merge pull requests this month:" . PHP_EOL . PHP_EOL;
```

**Comment by murrant** [comment]
Thanks for looking it over guys :)

**Comment by kkrumm1** [comment]
Why is this still open? LGTM

**Pull request merged**